tutorials/evals/evaluate_tool_calling.ipynb

```diff
@@ -33,12 +33,12 @@
    },
```

Comment:
<p>We might want to pin these requirements more loosely with <code>&gt;=</code> going forward. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Arize-ai/phoenix/pull/5675/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>